spec/acceptance/mod_php_spec.rb

```diff
@@ -121,19 +147,23 @@ class {'apache::mod::php':
       it 'succeeds in puppeting php' do
         apply_manifest(pp, catch_failures: true)
       end
-      if (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemmajrelease') == '16.04') ||
-         (fact('operatingsystem') == 'Debian' && fact('operatingsystemmajrelease') == '9')
-        describe file("#{$mod_dir}/php7.0.conf") do
-          it { is_expected.to contain '# somecontent' }
-        end
-      else
-        describe file("#{$mod_dir}/php5.conf") do
-          it { is_expected.to contain '# somecontent' }
-        end
+
+      path = if (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemmajrelease') == '16.04') ||
```

Comment:
Interesting that above it's a long if/else tree where here we define a path. Below it's duplicated. Perhaps it should be a `let(:php_config_path)` block?

Reply:
Swapped to a let

Reply:
didn't work either. Have just reverted it to how it was.